property/index.js

```diff
@@ -61,12 +62,13 @@ module.exports = yeoman.Base.extend({
 
   askForParameters: function() {
     this.name = this.options.propertyName;
-
+    
```

Comment:
Trailing whitespace, please remove. (This would be usually caught by eslint, but this project is not eslint-enabled yet.)